src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/IMessageReceiverProvider.java

```diff
@@ -0,0 +1,7 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.services.servicebus;
+
+import com.microsoft.azure.servicebus.IMessageReceiver;
+
+public interface IMessageReceiverProvider {
```

Comment:
Remove the `I` please, thats a .NET style thing

Reply:
is this interface just for tests? can we mock `ReceiverProvider`?

Reply:
yup it is ... this is so we can replace the connection to the SB

Reply:
@timja Well the thing is that it is providing an IMessageRetriever ... it not an interface really. Its the naming of the type of retriever ... lol
Il rename it anyway

Reply:
made it a messageReceiverFactory :D